src/Migrator.php

```diff
@@ -69,6 +70,7 @@ public static function migrate(): void
             M022UpdatePostRevisions::class,
             M023EnablePlanet4Blocks::class,
             M024RemoveNewIdentitySylesOption::class,
+            M025ReplaceDeprecatedColorsFromContent::class,
```

Comment:
Make sure to bump the migration prefix number since 1200cf93530ee9a9e0a85569644ac74617971d2f was merged in the meantime.